backend/src/main/java/com/step2020/server/managers/ActionManager.java

```diff
@@ -108,7 +108,13 @@ private void manageRequests(String requestId, Map<String, String> value) {
 	break;
       case 5:
 	eventManager.createEvent(requestId, value);
-	break;	
+	break;
+      case 6:	
+	eventManager.updateEvent(requestId, value);
+	break;
+      case 7:
```

Comment:
It'd be nice to use an enum for this code so that it's clear what each code does